docs/modules/deployment/pages/core/centos-rhel7/install-core.adoc

```diff
@@ -8,9 +8,9 @@ endif::[]
 
 .Install {page-component-title} with all built-in dependencies
 [source, console]
```

Comment:
remove [source,console]

and use ** 

`==` is used for headings not for commands.

Reply:
![Screenshot from 2024-10-21 14-27-02](https://github.com/user-attachments/assets/b7f6086d-0859-4a0e-97ad-bac682d4b4fa)
yes it resolved.